testing/scalatest.bzl

```diff
@@ -3,4 +3,4 @@ def scalatest_repositories():
     pass
 
 def scalatest_toolchain():
-    native.register_toolchain("//testing:scalatest_toolchain")
+    native.register_toolchains("@io_bazel_rules_scala//testing:scalatest_toolchain")
```

Comment:
Small question- your previous CI (IIRC) was without `@io_bazel_rules_scala` right? Why the change? What pushed it?

Reply:
It was a mistake - all toolchains have to be registered with external name otherwise they may be treated as different

Reply:
So it's precaution rather than something actually failed? (Ok with either, trying to understand which)

Reply:
I tried with the example repo - it doesn't even find the toolchain:
```
ERROR: Analysis of target '//example:example' failed; build aborted: invalid registered toolchain '//testing:scalatest_toolchain': no such package 'testing': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.

```

Reply:
Amazing. Great that we have this test workspace